justfile

```
@@ -67,6 +67,7 @@ bless: restart clean-test bless-insta-martin bless-insta-martin-core bless-insta
 # Bless the frontend tests
 [working-directory: 'martin/martin-ui']
 bless-frontend:
+    npm install
```

this will also update the lockfile.

How about we do this instead?

```suggestion
    npm clean-install
```